scripts/build.ps1

```diff
@@ -417,8 +423,13 @@ function CMake-Generate {
         }
         $Arguments += " -DCMAKE_BUILD_TYPE=" + $ConfigToBuild
     }
-    if ($DynamicCRT) {
-        $Arguments += " -DQUIC_STATIC_LINK_CRT=off -DQUIC_STATIC_LINK_PARTIAL_CRT=off"
+    if ($IsWindows) {
+        if ($DynamicCRT) {
+            $Arguments += " -DQUIC_STATIC_LINK_CRT=off -DQUIC_STATIC_LINK_PARTIAL_CRT=off"
+        }
+        if ($StaticCRT) {
```

Comment:
Why can't we statically link on Linux? Does CRT not exist?

Reply:
```
.PARAMETER DynamicCRT
    Builds msquic with dynamic C runtime (Windows-only).
```
I think no?

Reply:
Isn't CRT a windows thing? We link against glibc in linux?